pkg/k8sutil/pv.go

```diff
@@ -75,14 +77,26 @@ func UpdateReclaimPolicy(client *k8s.Clientset, pv *corev1.PersistentVolume) err
 	return err
 }
 
-func GetFlexVolumeName(pv *corev1.PersistentVolume) string {
+func GetVolumeName(pv *corev1.PersistentVolume) string {
 	// Rook creates rbd image with PV name
 	return pv.Name
 }
 
-func GetCSIVolumeName(pv *corev1.PersistentVolume) string {
-	// CSI created rbd image name
-	return pv.Spec.CSI.VolumeAttributes["imageName"]
+func WaitForPVCreation(pv *corev1.PersistentVolume) string {
```

Comment:
This is the one I suggested to change
```suggestion
func WaitForRBDImage(pv *corev1.PersistentVolume) string {
```